src/spatialdata/_core/_utils.py

```diff
@@ -0,0 +1,22 @@
+from spatialdata._core.spatialdata import SpatialData
+
+
+def find_common_table_keys(sdatas: list[SpatialData]) -> set[str]:
```

Comment:
should this function be private? just wondering, if the function is in a private module, then should it also be appended by `_` or is it redundant?

Reply:
yeah wasn't certain either, in a way it is redundant, but if someone imports it anyway and then another person would read that code they could think it is public. I am neutral here.

Reply:
Yeah I'd make it private.

Reply:
Resolved with [2576b62](https://github.com/scverse/spatialdata/pull/410/commits/2576b625056978ba94658fbbf7e24b8d84830b1c)